javalib/src/main/scala/java/util/PriorityQueue.scala

```diff
@@ -0,0 +1,125 @@
+package java.util
+
+import java.lang.Comparable
+import scala.math.Ordering
+import scala.collection.mutable
+import scala.annotation.tailrec
+import scala.language.existentials
+
+class PriorityQueue[E] protected (ordering: Ordering[_ >: E], _comparator: Comparator[_ >: E])
+    extends AbstractQueue[E]
+    with Serializable { self =>
+
+  def this(initialCapacity: Int) = {
+    this(defaultOrdering[E], null.asInstanceOf[Comparator[_ >: E]])
+    if (initialCapacity < 1) throw new IllegalArgumentException()
```

Comment:
For an `if` without `else`, which is therefore very imperative, the statement should be on the next line.